packages/techdocs-common/src/helpers.ts

```diff
@@ -43,21 +43,15 @@ export const parseReferenceAnnotation = (
     );
   }
 
-  // split on the first colon for the protocol and the rest after the first split
-  // is the location.
-  const [type, target] = annotation.split(/:(.+)/) as [
-    RemoteProtocol?,
-    string?,
-  ];
-
+  const { type, target } = parseLocationReference(annotation);
   if (!type || !target) {
```

Comment:
Does this exception still makes sense? Aren't the return values guarantee to be non null? Or is it a check for empty string, I think that that is already done for type.

Reply:
Myeah took a bit of a shortcut here I admit, cleaned it up a little. Yes it's for the empty string.